packages/terra-graphs/src/components/Bar/BarGraph.jsx

```diff
@@ -0,0 +1,47 @@
+/* eslint import/no-unresolved: off */
+import React from 'react';
+import PropTypes from 'prop-types';
+import Carbon from '@cerner/carbon-graphs/lib/js/carbon';
+import '../Graph.module.scss';
+import './BarGraph.module.scss';
+import utils from '@cerner/carbon-graphs/lib/js/helpers/utils';
+
+const propTypes = {
+  /**
+   * ID for graph created.
+   */
+  graphID: PropTypes.string.isRequired,
+  /**
+   * Graph configuration object
+   */
+  graphConfig: PropTypes.object.isRequired,
+  /**
+   * data to be displayed in graph
+   */
+  dataset: PropTypes.array,
+};
+
+const BarGraph = ({
+  graphConfig, dataset, graphID,
+}) => {
+  React.useEffect(() => {
+    const graph = Carbon.api.graph(graphConfig);
+    if (!(utils.isUndefined(dataset))) {
+      dataset.forEach((data) => {
+        graph.loadContent(Carbon.api.bar(data));
+      });
+    }
+  }, [graphConfig, dataset]);
+
+  return (
+    <div id="canvasContainer">
```

Comment:
do we expect developers to render multiple bar graphs? if so, we're going to have multiple components with the same canvas and tooltip ids. we may need to extract these into their own props

Reply:
Fixed [here](https://github.com/cerner/terra-graphs/pull/106/commits/8f1b7d01eb1f006f648f922b7b0ca25095b52fb2#diff-5e17603406b642035820d408335769ae48391021a7787fd86404b130590e9606R54).